code/game/objects/items.dm

```diff
@@ -632,3 +640,34 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		M.dirty++
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
+
+/obj/item/proc/update_force_string()
+	if(force)
+		switch(force)
+			if(1 to 3)
+				force_string = "very low"
+			if(4 to 6)
+				force_string = "low"
+			if(7 to 9)
+				force_string = "medium"
+			if(10 to 14)
+				force_string = "<font color=green>high</font>"
+			if(15 to 20)
+				force_string = "<font color=red>robust</combat>"
+			if(20 to 25)
+				force_string = "<font color=orange>very robust</font>"

```

Comment:
what do you mean? Isn't putting CSS in there technically hardcoding it as well/how do I do that?